tensorboard/webapp/plugins/npmi/views/annotations_list/annotation/annotation_component.ts

```diff
@@ -42,6 +42,7 @@ export class AnnotationComponent implements AfterViewInit, OnChanges {
   @Input() flaggedAnnotations!: string[];
   @Input() hiddenAnnotations!: string[];
   @Input() activeMetrics!: string[];
+  @Input() activeRuns!: string[];
```

Comment:
Is it necessary for the AnnotationComponent to be aware of the difference between active and inactive runs?  If the component never renders both at the same time in the UI, perhaps we can reduce the component's concerns.

An alternative to adding this field, is to just filter the `@Input() data` being passed into the component to only include data for the active runs.

Reply:
It actually already does only include data for active runs.

I thus only need this for the aforementioned svg height calculation and changed this to be numActiveRuns.